clients-simple/pom.xml

```diff
@@ -39,6 +43,24 @@
             <artifactId>common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.netflix.eureka</groupId>
+            <artifactId>eureka-client</artifactId>
+            <version>${eureka.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-netflix-eureka-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-openfeign</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.gchq.palisade</groupId>
+            <artifactId>readers-common</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
```

Comment:
I believe that this has been included because of the way that we add a serialiser to the data-service, this involves adding a DataFlavour to the request. Once the data-service uses the cache warming process then this process can be updated and the dependency can be removed.

Reply:
I don't think the clients actually use the addSerialiser call for the data-service.